src/no_pin.rs

```diff
@@ -0,0 +1,21 @@
+use embedded_hal::digital::v2::{OutputPin, PinState};
+
+use core::convert::Infallible;
+
+#[derive(Default)]
```

Comment:
could you add a basic documentation point here? just explain it's a dummy for the no-cs case.